numba_dppy/tests/kernel_tests/test_barrier.py

```diff
@@ -91,23 +90,21 @@ def test_local_memory(filter_str):
         pytest.skip()
     blocksize = 10
 
-    try:
-
-        @dppy.kernel("void(float32[::1])")
-        def reverse_array(A):
-            lm = dppy.local.array(shape=10, dtype=np.float32)
-            i = dppy.get_global_id(0)
-
-            # preload
-            lm[i] = A[i]
-            # barrier local or global will both work as we only have one work group
-            dppy.barrier(dppy.CLK_LOCAL_MEM_FENCE)  # local mem fence
-            # write
-            A[i] += lm[blocksize - 1 - i]
-
-    except:
+    if sys.platform in ["win32", "cygwin"] and filter_str == list_of_filter_strs[1]:
```

Comment:
pytest has `skipif` test decorator (https://docs.pytest.org/en/reorganize-docs/new-docs/user/skipping.html).
Moreover, it provides example for reusing this skips (https://docs.pytest.org/en/reorganize-docs/new-docs/user/skipping.html#re-using-skipif-decorators).
Reusing `skippers` eliminates code duplication and makes code more readable.